rector.php

```diff
@@ -43,7 +43,7 @@
     ]);
 
     $rectorConfig->sets([
-        SetList::PHP_83,
+        SetList::PHP_84,
         PHPUnitSetList::PHPUNIT_90,
         PHPUnitSetList::PHPUNIT_CODE_QUALITY,
         SymfonySetList::SYMFONY_71,
```

Comment:
off topic, but open for a PR for Symfony 7.2 ?

Reply:
Of course ! https://github.com/OskarStark/doctor-rst/pull/1887